src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc

```diff
@@ -38,10 +38,19 @@
 #include "src/core/tsi/alts/handshaker/alts_handshaker_client.h"
 #include "src/core/tsi/alts/handshaker/alts_shared_resource.h"
 #include "src/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector.h"
+#include "src/core/util/env.h"
 #include "src/core/util/memory.h"
 #include "src/core/util/sync.h"
 #include "upb/mem/arena.hpp"
 
+#define GRPC_EXPERIMENTAL_ALTS_HANDSHAKER_KEEPALIVE_PARAMS \
```

Comment:
Please make this a constant (rather than a #define) and also please wrap the constants in an anonymous namespace.

Reply:
Done

Reply:
It seems like this did not occur? Did you forget to push the commit by any chance?

Reply:
My bad, didn't push the commit